docs/ROADMAP.md

```diff
@@ -49,45 +49,59 @@ Non-Technical Requirements for v0.1:
   * License Agreement
 * Account information/access from WHO to submit app on behalf of their accounts (also needed for code signing) ([#20](https://github.com/WorldHealthOrganization/app/issues/20) :warning:)
```

Comment:
Remove as this isn't what happened.
```suggestion
```

Reply:
Future plan is to setup Fastlane and have that handle app store submission